### PR TITLE
test backup requirement 

### PR DESCRIPTION
**Pull request by abellj01**
Just completed test cases for backup #8

### CODE REVIEW

File: backup.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
+def backup(answer) :
```
Comment (sebastianovide) [inline backup.py]: this function is overwritten by  L33

File: backup.py [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    >>> backup('Yes')
+    'Pass'
+    >>> backup('Y')
+    'Pass'
+    >>> backup('y')
+    'Pass'
+    """
+    return "Pass"
+
+def backup(answer) :
```
Comment (sebastianovide) [inline backup.py]: this function is overwritten by L33

File: backup.py [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+    >>> backup(' ')
+    'Fail'
+    >>> backup("'")
+    'Fail'
+    >>> backup('"')
+    'Fail'
+    """
+    return "Fail"
+
+def backup(answer) :
```
Comment (sebastianovide) [inline backup.py]: embed the previous two function in this one

Comment (abellj01) [reply backup.py]: Done in last pull request

File: backup.py (hunk 2 above)
Comment (abellj01) [reply backup.py]: Done in last pull request

**Pull request merged**